templates/repo/issue/view_content/pull.tmpl

```diff
@@ -167,9 +167,11 @@
 				<div class="item text">
 					{{svg "octicon-x"}}
 					{{$.locale.Tr "repo.pulls.files_conflicted"}}
-					{{range .ConflictedFiles}}
-						<div>{{.}}</div>
-					{{end}}
+					<ul>
+						{{range .ConflictedFiles}}
+							<li>{{.}}</li>
+						{{end}}
+					</ul>
```

Comment:
Exactly because of that? We use three times the same code, the only difference is what we range over.
That's exactly why I think it would be a good idea.

Reply:
I do not think extracting too simple code is a good idea.

Before, there were many "common" templates , for example, the search filter. In the end, no code uses these common templates any more.

Google says:

> Over abstraction can create unmaintainable, untestable monstrosities. Under abstraction may mean duplicate code, but duplicate code is almost always better than unmaintainable code. That is until it becomes unmaintainable itself. Duplicate code can be fixed via factoring; the over-abstracted code may not be fixable.

Reply:
The example is: `base/search.tmpl`.

In history, many templates called it, some people thought that "oh, the code is extracted here and it's clear"

However, now, only one legacy template calls it. Because it's over abstraction